core/src/main/scala/org/http4s/Compression.scala

```diff
@@ -0,0 +1,93 @@
+package org.http4s
```

Comment:
We've generally reserved this package for core HTTP types.  Supporting code usually lives somewhere else.  Maybe `org.http4s.util`?